lavamoat/node/policy-override.json

```diff
@@ -20,6 +12,34 @@
         "@babel/plugin-proposal-nullish-coalescing-operator": true
       }
     },
+    "@eslint/eslintrc": {
+      "packages": {
+        "@babel/eslint-parser": true,
+        "@babel/eslint-plugin": true,
+        "@metamask/eslint-config": true,
+        "@metamask/eslint-config-nodejs": true,
+        "eslint": true,
+        "eslint-config-prettier": true,
+        "eslint-plugin-import": true,
+        "eslint-plugin-node": true,
+        "eslint-plugin-prettier": true,
+        "eslint-plugin-react": true,
+        "eslint-plugin-react-hooks": true
+      }
+    },
+    "eslint-module-utils": {
+      "packages": {
+        "eslint-import-resolver-node": true
+      }
+    },
```

Comment:
For some reason this is only used by `eslint-module-utils` with `--build-type flask` or `beta`.